source/Plugins/Process/Dpu/DpuRank.cpp

```diff
@@ -78,10 +56,19 @@ bool DpuRank::Open(llvm::StringRef profile) {
   } else if (args_split.first.equals("simulator")) {
```

Comment:
do we still the  args_split = profile.split('/') above ?
we need to change this syntax too ?

Reply:
We may want to change that. I don't know what @rjodinupmem thinks about it.